e2e-test/reconcile_namespace_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eo pipefail
+
+test_dir=e2e-test
+
+source ${test_dir}/functions.sh
```

Comment:
Always quote paths that contain variables in case a variable contains a space.
```suggestion
source "${test_dir}/functions.sh"
```